README.md

```diff
@@ -93,7 +93,7 @@ The script uses [anti-captcha.com](https://anti-captcha.com/) service to bypass
   **cron** setup (more: https://help.ubuntu.com/community/CronHowto) :
   
   ```
-  $ sudo crontab -e
+  $ crontab -e

```

Comment:
There is no need to run crontab as sudo user, an ordinary user should be enough.

Reply:
Good point, approved :) !